src/main/resources/db/migration/V009__Add_pending_notification_column_and_index_to_envelopes.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE envelopes
+ADD COLUMN pending_notification BOOLEAN DEFAULT FALSE;
+
+CREATE INDEX envelopes_pending_notification_envelope_id_idx ON envelopes (id, pending_notification);
```

Comment:
why on those 2 columns?

Reply:
yeah, envelopes `id` is primary key so just `pending_notification` is enough. Updated the script